start.sh

```diff
@@ -5,4 +5,28 @@ if [ "$ENABLE_KERBEROS" = "yes" ]; then
   export DIVOLTE_JAVA_OPTS="-Djava.security.krb5.conf=/etc/krb5.conf -Dsun.security.krb5.debug=true"
 fi
 
+if [ -z "$DIVOLTE_HDFS_SINK_WORKING_DIR" ]; then
+	# No specific dir set use default and create it if not exists
+	if [ ! -d /tmp/working ]; then
+		mkdir -p /tmp/working
+	fi
+else
+	#make sure dir exists
+	if [ ! -d "$DIVOLTE_HDFS_SINK_WORKING_DIR" ]; then
+		mkdir -p "$DIVOLTE_HDFS_SINK_WORKING_DIR"
```

Comment:
The `mkdir -p "$DIVOLTE_HDFS_SINK_WORKING_DIR"` might be nicer than the `mkdir -p /tmp/working` above.

I would change it to:
```
DIVOLTE_HDFS_SINK_WORKING_DIR=${DIVOLTE_HDFS_SINK_WORKING_DIR:/tmp/working}
mkdir -p "$DIVOLTE_HDFS_SINK_WORKING_DIR"
```
But this is a matter of taste I'd say.

Reply:
I like your taste. Done like this....